ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml

```diff
@@ -34,12 +34,15 @@
     - Freemarker
     - Freetype
     - cmake
+    - Make-3.82
```

Comment:
Is the make installed in the cygwin role not adequate?

Reply:
to build openjdk8 it needs an older version of Make (not make 4)